libs/ledger-live-common/src/wallet-api/types.ts

```diff
@@ -14,6 +14,7 @@ export type {
 } from "@ledgerhq/wallet-api-core";
 
 export {
+  FAMILIES as WalletAPI_FAMILIES,
```

Comment:
`FAMILIES` is not a type. IMO it should not be put in the `types.ts` file to avoid confusion. It could be moved to the `constants.ts` file, since it's more akin to a constant (it's fixed for a given version of the wallet-api).
Otherwise it could also be moved to a new `common.ts` file if it's not considered a constant.

Reply:
Good catch, I'll move that